lib/lint.js

```diff
@@ -84,7 +84,16 @@ function lintJavascript(paths = []) {
   paths = paths.filter(path => fs.existsSync(path));
 
   let parentNodePath = path.resolve(path.join(path.basename(__filename), '../node_modules'));
-  let eslintPath = require.resolve('eslint/bin/eslint', { paths: [parentNodePath] });
+
+  let eslintPath;
+  try {
```

Comment:
Could you explain why we need this?

Reply:
Somehow, after upgrading to ESLint 8.x, the old resolution logic does not work anymore. I don't know if the eslint have changed the path, or what - but the binary is present at different location.

The try..catch here is allowing us to use either old (7.x) or new (8.x) eslint, by trying different paths to the binary.

ps: if you know of a cleaner way to achieve the same - i am open for suggestions!